priv/repo/migrations/20170317153551_create_message.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Healthlocker.Repo.Migrations.CreateMessage do
+  use Ecto.Migration
+
+  def change do
+    create table(:messages) do
+      add :body, :string, null: false
+      add :name, :string
```

Comment:
[`preload`](https://hexdocs.pm/ecto/Ecto.Repo.html#c:preload/3) should handle this for us. We could also combine it with a left join to only have one query for the whole thing. Wanna screenshare tomorrow on this?